### PR TITLE
play/defensive_clear mypy fix

### PR DESCRIPTION
**Pull request by oddlyspookycherry**
## Description
Fixed mypy typing errors in the file play/defensive_clear.

## Steps to test
Find "Pull request checks / mypy" and click details.

Expected result: There should be no mypy errors for play/defensive_clear

### CODE REVIEW

File: rj_gameplay/rj_gameplay/play/defensive_clear.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ def tick(
         world_state: rc.WorldState,
         prev_results: role.assignment.FlatRoleResults,
         props,
-    ) -> Tuple[Dict[Type[tactic.SkillEntry], List[role.RoleRequest]], List[tactic.SkillEntry]]:
+    ) -> Tuple[Dict[SkillEntry[Any], List[RoleResult]], Any]:
```
Comment (oddlyspookycherry) [inline rj_gameplay/rj_gameplay/play/defensive_clear.py]: Function had incorrect return type. Changed.

**Review by kfu02 — CHANGES_REQUESTED** [review]
Fix the style errors. (Not your fault for not seeing; the checks take a while to run after you submit a PR.) 

Also, is there any more specific type than Any that could fix the return type? In general Any should be avoided because it's basically the same as not giving a type at all.

File: rj_gameplay/rj_gameplay/play/defensive_clear.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 from stp.role.assignment.naive import NaiveRoleAssignment
 import stp.rc as rc
 import numpy as np
-from typing import Any, Dict, Generic, Iterator, List, Optional, Tuple, Type, TypeVar
+from typing import Dict, List, Tuple
```
Comment (oddlyspookycherry) [inline rj_gameplay/rj_gameplay/play/defensive_clear.py:11]: Removed unused imports.

File: rj_gameplay/rj_gameplay/play/defensive_clear.py [hunk 3]
```diff
@@ -25,7 +25,8 @@ def tick(
         world_state: rc.WorldState,
         prev_results: role.assignment.FlatRoleResults,
         props,
-    ) -> Tuple[Dict[tactic.SkillEntry[Any], List[role.RoleResult]], Any]:
+    ) -> Tuple[Dict[tactic.SkillEntry, List[role.RoleResult]],
```
Comment (oddlyspookycherry) [inline rj_gameplay/rj_gameplay/play/defensive_clear.py:28]: Return type changed to a proper one.

File: rj_gameplay/rj_gameplay/play/defensive_clear.py [hunk 4]
```diff
@@ -43,6 +44,7 @@ def tick(
         role_results = play.unflatten_results(flat_results)
 
         # Get list of all skills with assigned roles from tactics
+        skills: List[tactic.SkillEntry] = []
         skills = self.two_mark.tick(world_state, role_results[self.two_mark])
```
Comment (kfu02) [inline rj_gameplay/rj_gameplay/play/defensive_clear.py]:
```suggestion
        skills += self.two_mark.tick(world_state, role_results[self.two_mark])
```
line 47 inits an empty list of SkillEntries but then this line resets it to the return type of a tactic's tick(), should change to += instead

**Review by kfu02 — APPROVED** [review]
Works well on my machine, nice job. One more minor change in the defensive_clear.py file but otherwise LGTM.

**Review by adithya-gv — APPROVED** [review]
LGTM!

**Pull request merged**